src/diffusion_for_multi_scale_molecular_dynamics/models/diffusion_mace.py

```diff
@@ -53,10 +84,12 @@ def input_to_diffusion_mace(
 
     # node features are int corresponding to atom type
     # TODO handle different atom types
-    atom_types = torch.zeros(batch_size * n_atom_per_graph)
-    node_attrs = torch.nn.functional.one_hot(atom_types.long(), num_classes=1).to(
-        atom_types
-    )
+    atom_types = batch[NOISY_AXL].A
+    node_attrs = (
```

Comment:
I think there is a utility function somewhere that creates one-hots. It would be better to call that function here.